changelog/unreleased/features/2096-partition-timeout.md

```diff
@@ -0,0 +1,4 @@
+VAST now stores its partitions per layout. The new option
```

Comment:
```suggestion
VAST now stores its partitions per layout.
```

This is hard to comprehend. Can we say something like "VAST now creates one partition per layout"?